app/presenters/queries/content_item_presenter.rb

```diff
@@ -0,0 +1,24 @@
+module Presenters
+  module Queries
+    class ContentItemPresenter
+      def self.present(content_item)
+        version = Version.find_by(target: content_item)
+        new(content_item, version).present
+      end
+
+      def initialize(content_item, version)
+        self.content_item = content_item
+        self.version = version
+      end
+
+      def present
+        content_item.as_json.deep_symbolize_keys
```

Comment:
@elliotcm from what I can see the concern `SymbolizeJSON` will call `deep_symbolize_keys` on Hash and Array values but doesn't symbolize top level keys, is my reading of this right? I've removed the `deep_symbolize_keys` and replaced with `symbolize_keys`. Without this top level keys are strings.